pt/lessons/advanced/error-handling.md

```diff
@@ -1,19 +1,19 @@
 ---
-version: 0.9.0
+version: 1.0.0
 title: Tratamento de Erros
 ---
 
-Embora seja mais comum o retorno da tupla `{: erro, reason}`, Elixir suporta exceções e nesta lição veremos como lidar com erros e os diferentes mecanismos disponíveis para nós.
+Embora seja mais comum o retorno da tupla `{:error, reason}`, o Elixir suporta exceções e nesta lição veremos como lidar com erros e os diferentes mecanismos disponíveis para nós.
 
-Em geral, a convenção em Elixir é criar uma função (`example/1`) que retorna` {: ok, result} `e` {: error, reason} `e uma função separada (`example!/1`) que retorna o `result` desembrulhado ou gerará um erro.
+Em geral, a convenção em Elixir é criar uma função (`example/1`) que retorna` {:ok, result} ` e ` {:error, reason} ` e uma função separada (`example!/1`) que retorna o `result` desempacotado ou levanta um erro.
```

Comment:
I'm being a bit picky picky here 😄 but we could also remove spaces between the apostrophe and curly brackets ` {:ok, result} ` and here ` {:error, reason} `.